server/test/integration/com/thoughtworks/go/server/service/AutoTriggerDependencyResolutionTest.java

```diff
@@ -90,6 +90,10 @@
     private GoConfigFileHelper configHelper = new GoConfigFileHelper();
     private ScheduleTestUtil u;
 
+    static {
+        new SystemEnvironment().setProperty("dependency.material.check.threads", "0");
```

Comment:
Hmm. Do you need to tear this change down or revert it somewhere, after the test has run? Could affect other tests.